src/Component/Nav/CategoriesNav.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const categoryData = [
+  "Sustainability",
+  "Wallpapers",
+  "COVID-19",
+  "Travel",
+  "Nature",
+  "Textures & Patterns",
+  "Current Events",
+  "People",
+  "Business & Work",
+  "Technology",
+  "Animals",
+  "Interiors",
+  "Architecture",
+  "Food & Drink",
+  "Athletics",
+  "Spirituality",
+  "Health & Wellness",
+  "Film",
+  "Fasion",
+  "Experimental",
+  "Arts & Culture",
+  "History",
+];
+
+const editorialData = ["Editorial", "Following"];
+
+const Category = (props) => {
+  return (
+    <Link>
+      <li>{props.category}</li>
+    </Link>
+  );
+};
+
+const CategoriesNav = () => {
+  return (
+    <CategoryContainer>
+      <Left>
+        <ul>
+          {editorialData.map((eachCategory) => {
+            return <Category category={eachCategory} />;
+          })}
+        </ul>
+      </Left>
+      <div className="border" />
+      <Categories>
+        <div>
+          <ul>
+            {categoryData.map((eachCategory) => {
+              return <Category category={eachCategory} />;
+            })}
+          </ul>
+        </div>
+      </Categories>
+      <Right>
+        <div className="gradient" />
+        <ul>
+          <Category category={["view all"]} />
+        </ul>
+      </Right>
+    </CategoryContainer>
+  );
+};
+
+export default CategoriesNav;
+
+const CategoryContainer = styled.div`
+  height: 62px;
+  display: flex;
+  background-color: #fff;
+
+  .border {
+    background-color: #d1d1d1;
+    width: 1px;
+    margin-top: 12px;
+    margin-bottom: 12px;
+    flex-shrink: 0;
+  }
+
+  a {
+    font-size: 14px;
+    color: #767676;
+    &:hover {
+      color: #111;
+    }
+  }
+`;
+
+const Left = styled.div`
+  ${(props) => props.theme.flexCenter};
```

Comment:
```suggestion
  ${({ theme }) => theme.flexCenter};
```
함수 파라미터에서 곧바로 구조분해 할당 가능합니다